modules/ROOT/partials/misc/url-handling.adoc

```diff
@@ -2,7 +2,7 @@
 
 === Relative URLs
 
-This will convert all URLs within the same domain to relative URLs. The URLs will be relative from the <<document_base_url,document_base_url>>.
+This will convert all URLs within the same domain to relative URLs. The URLs will be relative from the xref:document_base_url[document_base_url].
```

Comment:
```suggestion
This will convert all URLs within the same domain to relative URLs. The URLs will be relative from the xref:configure/url-handling.adoc#document_base_url[document_base_url].
```

Reply:
done